torch/csrc/autograd/VariableTypeManual.cpp

```diff
@@ -214,7 +214,7 @@ Tensor& copy_(
       if (!self.is_same_size(src_fw_grad)) {
         new_fw_grad = src_fw_grad.broadcast_to(self.sizes());
       } else {
-        new_fw_grad = src_fw_grad;
+        new_fw_grad = src_fw_grad.clone();
```

Comment:
This is the same situation as https://github.com/pytorch/pytorch/pull/79664#discussion_r899198907 right? If so I'll defer to what the outcome of that discussion is